src/devices/Max31865/Max31865.cs

```diff
@@ -0,0 +1,290 @@
+        /// <param name="shouldDispose">True to dispose the SPI device</param>
+        public Max31865(SpiDevice spiDevice, PlatinumResistanceThermometerType platinumResistanceThermometerType, ResistanceTemperatureDetectorWires resistanceTemperatureDetectorWires, ElectricResistance referenceResistor, ConversionFilterMode filterMode = ConversionFilterMode.SixtyHz, bool shouldDispose = true)
+        {
+            _spiDevice = spiDevice ?? throw new ArgumentNullException(nameof(spiDevice));
+            _prtType = platinumResistanceThermometerType;
+            _rtdWires = resistanceTemperatureDetectorWires;
+            _filterMode = filterMode;
+            _filterModeSleep = filterMode == ConversionFilterMode.FiftyHz ? FilterMode50HzSleep : FilterMode60HzSleep;
+            _referenceResistor = referenceResistor.Ohms;
+            _shouldDispose = shouldDispose;
+            Initialize();
+        }
+
+        /// <inheritdoc/>
+        public void Dispose()
+        {
+            if (_shouldDispose)
+            {
+                _spiDevice?.Dispose();
+            }
+
+            _spiDevice = null!;
+        }
+
+        /// <summary>
+        /// The fault state of the sensor
+        /// </summary>
+        public FaultStatus Faults
+        {
+            get
+            {
+                Span<byte> readBuffer = stackalloc byte[2];
+                WriteRead(Register.FaultSatus, readBuffer);
+
+                return new FaultStatus(
+                    (readBuffer[1] & 0x04) == 0x04,
+                    (readBuffer[1] & 0x08) == 0x08,
+                    (readBuffer[1] & 0x20) == 0x20,
+                    (readBuffer[1] & 0x10) == 0x10,
+                    (readBuffer[1] & 0x40) == 0x40,
+                    (readBuffer[1] & 0x80) == 0x80);
+            }
+        }
+
+        /// <summary>
+        /// Standard initialization routine.
+        /// </summary>
+        /// <remarks>
+        /// You can add new write lines if you want to alter the settings of the device. Settings can be found in the Technical Manual
+        /// </remarks>
+        private void Initialize()
+        {
+            ReadOnlySpan<byte> configurationSetting = stackalloc byte[]
+            {
+                (byte)Register.ConfigurationWrite,
+                (byte)((byte)(_rtdWires == ResistanceTemperatureDetectorWires.ThreeWire ? Configuration.ThreeWire : Configuration.TwoFourWire) | (byte)(_filterMode == ConversionFilterMode.FiftyHz ? Configuration.Filter50HZ : Configuration.Filter60HZ))
+            };
+
+            Write(configurationSetting);
+        }
+
+        /// <summary>
+        /// Clears all the faults
+        /// </summary>
+        private void ClearFaults()
+        {
+            Span<byte> configuration = stackalloc byte[2];
+            WriteRead(Register.ConfigurationRead, configuration);
+
+            configuration[1] = (byte)(configuration[1] & ~0x2C);
```

Comment:
for constants like this, used in the code, you may want to add a comment from which page in the documentation they are coming from. It does help for maintenance of the code and debug.
You can point at the beginning of the class the url where the doc is for example and then just point out the page or the section.